### PR TITLE
Portfolio: accessibility

### PR DESCRIPTION
**Pull request by ZuhRa-HashiMi**
- Page titles - Brought changes to the page title.
- Image text alternatives** (alt text) - No issues found.
- Text headings - No issues found.
- Resize - Increased font-size and font-weight for disabled readers to easily see most information being displayed on their screen and Increasing the font-size of the about me section.
- Color contrast - Change the hover color on the buttons to be clearer and more visible by readers.
- Interaction (labels are optional but recommended as voiceover tools can read them, and should be set as invisible to match the Figma design) - No issues found
- Moving content - No issues found
- Multimedia - No issues found
- The basic structure of the page - No issues found

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -278,8 +278,8 @@ button:enabled {
 }
 
 button:hover {
-  border: 1px solid #f55b00;
-  background-color: #fff5e1;
+  border: 1px solid #f85b00;
+  background-color: #fff;
   color: #f55800;
 }
```
Comment (XRichardroidX) [inline css/style.css:284]: Change the hover color to look visible for readers

File: css/style.css [hunk 2]
```diff
@@ -197,9 +197,9 @@ a:hover {
 .recent-works h3 {
   text-align: center;
   border-bottom: #000 3px;
-  font-size: 40px;
+  font-size: 60px;
   font-style: normal;
-  font-weight: 400;
+  font-weight: 600;
```
Comment (XRichardroidX) [inline css/style.css:202]: Increased font-size and font-weight for disabled readers to easily see most information being displayed on their screen.

File: index.html [hunk 3]
```diff
@@ -12,7 +12,7 @@
     <link rel="preconnect" href="https://fonts.googleapis.com">
   
       
-    <title>Zuhra Hashimi - Portfolio</title>
+    <title>Zuhra Hashimi_Portfolio</title>
```
Comment (XRichardroidX) [inline index.html:15]: Making the title clearer for readers.

**Review by XRichardroidX — COMMENTED** [review]
There was no problem and this website is more accessible, We brought some changes for more accessibility to our readers 🙂👍.

File: css/style.css (hunk 2 above)
Comment (topeogunleye) [reply css/style.css:202]: image

## As written in the project requirements. I kindly suggest that you ensure that you:
- [ ] Add your report to the description of your pull request.
- [ ] List all points that you have checked, Example:

  - Page titles - No issues found
  - Image text alternatives** `(alt text)` - No issues found
  - Text headings - No issues found
  - Color contrast - No issues found
  - Resize - No issues found
  - Interaction (labels are optional but recommended as voiceover tools can read them, and should be set as invisible to match the Figma design) - No issues found
  - Moving content - No issues found
  - Multimedia - No issues found
  - The basic structure of the page - No issues found

**Review by topeogunleye — CHANGES_REQUESTED** [review]
Hi @ZuhRa-HashiMi, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

Try-Again

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- Nothing to mention

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @topeogunleye in your question so I can receive the notification.


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your @topeogunleye so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by 1GORDON — APPROVED** [review]
# Status: APPROVED 🟢 

Hi @ZuhRa-HashiMi ✋

Your project is complete!  There is nothing else to say other than... it's time to merge it: ship it:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**